docs/source/recipes/plot_13_recipe.py

```diff
@@ -0,0 +1,265 @@
+"""
+Calculate and plot the Niño 3.4 Index
+====================================
+
+In this recipe, we will calculate and plot the sea surface temperature (SST)
+anomaly in the Niño 3.4 region. According to `NCAR Climate Data Guide
+<https://climatedataguide.ucar.edu/climate-data/nino-sst-indices-nino-12-3-34-4-oni-and-tni>`_,
+the Niño 3.4 anomalies may be thought of as representing the average equatorial
+SSTs across the Pacific from about the dateline to the South American coast.
+The Niño 3.4 index typically uses a 5-month running mean, and El Niño or La
+Niña events are defined when the Niño 3.4 SSTs exceed +/- 0.4 degrees C for a
```

Comment:
```suggestion
Niña events are defined when the Niño 3.4 SSTs exceed +/- 0.4 degrees Celsius for a
```

or °C as a symbol. ideally, since 'degrees C' is kind of inbetween the shorthand and longhand.